roles/idrac_bios/molecule/update_attributes_immediate_with_jobwait/cleanup.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Converge file to update attributes with apply time as Immediate
+  hosts: all
+  gather_facts: false
+  vars:
+    idrac_ip: "{{ lookup('env', 'IDRAC_IP') }}"
```

Comment:
check if vars needed in file

Reply:
Reviewed and removed unnecessary variables